collectors/VMStatsCollector.py

```diff
@@ -0,0 +1,44 @@
+from BaseCollector import BaseCollector
+import os
+from prometheus_client.core import GaugeMetricFamily
+from tools.Resources import Resources
+from tools.YamlRead import YamlRead
+
+
+class VMStatsCollector(BaseCollector):
+    def __init__(self):
+        self.wait_for_inventory_data()
+        self.statkey_yaml = YamlRead('collectors/statkey.yaml').run()
+        self.g = GaugeMetricFamily('vrops_vms_stats', 'testtext', labels=['vccluster', 'datacenter', 'virtualmachine', 'hostsystem', 'statkey'])
```

Comment:
could you please rename this to `vrops_vm_stats`?
then we have 
* `vrops_vm_stats`
* `vrops_vm_properties`

Reply:
Done !